src/providers/personalData/store.tsx

```diff
@@ -107,13 +107,15 @@ export const usePersonalDataStore = createStore<PersonalDataStoreState, Personal
     persist: {
       key: 'personalData',
       whitelist: WHITELIST,
-      version: 0,
+      version: 0.1,
       onRehydrateStorage: () => {
         WHITELIST.forEach((item) => {
           window.localStorage.removeItem(item)
         })
       },
-      migrate: () => null,
+      migrate: (oldState) => ({
+        recentSearches: oldState.filter((item: RecentSearch) => !Object.prototype.hasOwnProperty.call(item, 'type')),
```

Comment:
Old state here isn't an array but an object. If this migration is run, an exception will be thrown: `Failed to migrate store "personalData": e.filter is not a function`. You can test it out yourself by getting some recent searches on deployment with old search, use export state function from developer overlay and then import into app running those changes and refresh